src/freertos_drivers/net_cc32xx/CC32xxWiFi.cxx

```diff
@@ -456,6 +527,95 @@ void CC32xxWiFi::wlan_mac(uint8_t mac[6])
     sl_NetCfgGet(SL_NETCFG_MAC_ADDRESS_GET, nullptr, &len, mac);
 }
 
+/*
+ * CC32xxWiFi::test_mode_start()
+ */
+void CC32xxWiFi::test_mode_start()
+{
+    /* note, sl_Task MUST be a pthread because the Wi-Fi task uses the pthread
+     * interface such as pthread_self().
+     */
+    pthread_t thread;
+    pthread_attr_t attr;
+    struct sched_param sched_param;
+
+    pthread_attr_init(&attr);
+    sched_param.sched_priority = sched_get_priority_max(SCHED_FIFO);
+    pthread_attr_setschedparam(&attr, &sched_param);
+    pthread_attr_setstacksize(&attr, 2048);
+    pthread_create(&thread, &attr, sl_Task, nullptr);
```

Comment:
i thought we moved all of os_thread_create to this code by now on the cc3220?

Reply:
at the time I started this, we did not have pthread merged back yet.  Now that we do, I made the change over to os_thread_create().